app/assets/javascripts/listings/ListingController.js.coffee

```diff
@@ -206,8 +206,28 @@ ListingController = (
   $scope.hasMultipleAMIUnits = ->
     _.keys($scope.listing.groupedUnits).length > 1
 
+  $scope.occupancy = (unitSummary) ->
+    return '1' if unitSummary.maxOccupancy == 1
+    unitSummary.minOccupancy + '-' + unitSummary.maxOccupancy
+
+  $scope.occupancyUnit = (maxOccupancy) ->
+    return 'person' if maxOccupancy == 1
+    'people'
+
+  $scope.formatBaths = (numberOfBathrooms) ->
+    return 'Shared' if numberOfBathrooms == 0
+    return '1/2 bath' if numberOfBathrooms == 0.5
+
+    fullBaths = Math.floor numberOfBathrooms
+    andAHalf = numberOfBathrooms - fullBaths == 0.5
```

Comment:
@mluedke2 If you were feeling clever, you could do `andAHalf = numberOfBathrooms / .5 % 2`, but what you have is probably easier to read!